static/js/jira-select-github-version.js

```diff
@@ -0,0 +1,17 @@
+$('.jiraSelectGitHubVersion__options__card').click(function (event) {
+	event.preventDefault();
+	const selectedCard = event.target.className;
```

Comment:
maybe we could just do `$('.jiraSelectGitHubVersion__options__card').removeClass('selected')`
and
`$(event.target).addClass('selected')`

------

`selectedCard.substring(selectedCard.indexOf(' ') + 1)` seems not very robust to me as it depends on the order or class names.

Reply:
Good point.

Reply:
How about `$(event.target).closest('.jiraSelectGitHubVersion__options__card').addClass('selected')`?
I think above may allow us to remove the whole block bellow:
```javascript
        const selectedVersion = selectedCard.substring(selectedCard.indexOf(' ') + 1);
	const className = `.jiraSelectGitHubVersion__options__card.${selectedVersion}`;

	if (selectedVersion === "cloud") {
		$(".jiraSelectGitHubVersion__options__card.server").removeClass("selected");
	}

	if (selectedVersion === "server") {
		$(".jiraSelectGitHubVersion__options__card.cloud").removeClass("selected");
	}

	$(`${className}`).addClass("selected");
```

Reply:
That looks like it would select the correct element but we would still need to remove it from the other card if the user than selects the other option. How would this help us remove the conditional block?

Reply:
I meant we could first remove the `'.selected'` from both of the `'.jiraSelectGitHubVersion__options__card'`, then add `'.selected'` only to the `$(target).closest('.jiraSelectGitHubVersion__options__card')`
---
But having say that, I am happy to leave the code as it is. I don't see a big problem here.

Reply:
Did you see the most recent change?

Reply:
🤦 realised I forgot to push them.

Reply:
Oh, haha, all good. Now much better :) 
ps: And I forgot the approve it again, sorry.